src/note-API/auth-API.js

```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+
+const loginAPI = async (user) => {
+  return axios.post("/api/auth/login", user);
+}
+
+const signUpAPI = (user) => {
+  return axios.post("/api/auth/signup", user);
+};
```

Comment:
You are using async but did not use await also putting this in a try-catch block will be better.